mappings/net/minecraft/client/render/debug/StructureDebugRenderer.mapping

```diff
@@ -1 +1,12 @@
 CLASS net/minecraft/class_870 net/minecraft/client/render/debug/StructureDebugRenderer
+	FIELD field_4624 client Lnet/minecraft/class_310;
+	FIELD field_4625 structureAlliances Ljava/util/Map;
+	FIELD field_4626 structures Ljava/util/Map;
+	FIELD field_4627 structureBoxes Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/class_310;)V
+		ARG 1 client
+	METHOD method_3871 addStructureData (Lnet/minecraft/class_3341;Ljava/util/List;Ljava/util/List;Lnet/minecraft/class_2874;)V
```

Comment:
`addStructure` could also just work.

Reply:
Except this class doesn't directly deal with structures. It deals with information of _where_ structures are and the only information it really has is "green blob here", "red blob there".